educhain/models/content_models.py

```
@@ -417,3 +417,96 @@ def show(self):
+            print(f"   Content: {segment.content}")
+        
+        print("\nCONCLUSION:")
+        print(self.conclusion)
+        
+        if self.key_takeaways:
+            print("\nKEY TAKEAWAYS:")
+            for takeaway in self.key_takeaways:
+                print(f"- {takeaway}")
+        
+        if self.call_to_action:
+            print(f"\nCALL TO ACTION:")
```

Ruff (0.14.7)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)